README.dev.md

```diff
@@ -30,9 +30,10 @@ tools installed.
 # unit tests with mocked representations of repository behavior
 pytest
 pytest tests/
-
 ```
 
+This run will also report the percentage of code covered by the tests.
```

Comment:
Maybe point to coverage directory and the README that is already in it

Reply:
Good point: done!